ckanext/cnra_schema/templates/scheming/package/snippets/package_form.html

```diff
@@ -1,9 +1,54 @@
 {% ckan_extends %}
 
+{% block basic_fields %}
+  {% block styles %}
+      <link rel="stylesheet" href="/css/additional_metadata_form.css" />
+  {% endblock%}
+
+  <p class=right-text>
+    Questions? See: <a href="/pages/metadata"><u>Metadata Guide</u></a>
+  </p>
+  
+  {%- if not dataset_type -%}
+    <p>
+    dataset_type not passed to template. your version of CKAN
+    might not be compatible with ckanext-scheming
+    </p>
+  {%- endif -%}
+  {%- set schema = h.scheming_get_dataset_schema(dataset_type) -%}
+  <!-- This for loop is intended to display the primary metadata. More details can be found on ticket ESN-1341-->
```

Comment:
Another suggestion is to use jinja comments `{# #}` instead of html comments `<!-- -->`. Jinja comments won't be rendered and users won't see it in html code.